ui/raidboss/data/06-ew/raid/p1s.ts

```diff
@@ -48,6 +77,32 @@ const triggerSet: TriggerSet<Data> = {
       netRegexJa: NetRegexes.startsUsing({ id: '662A', source: 'エリクトニオス', capture: false }),
       response: Responses.aoe(),
     },
+    {
+      id: 'P1S Shackles of Companionship',
+      type: 'GainsEffect',
+      netRegex: NetRegexes.gainsEffect({ effectId: 'AB6' }),
+      condition: Conditions.targetIsYou(),
+      durationSeconds: (_data, matches) => parseFloat(matches.duration),
+      infoText: (_data, _matches, output) => output.text!(),
+      outputStrings: {
+        text: {
+          en: 'Close Shackles on YOU',
```

Comment:
Do you think it's worth calling these out to folks other than the target, since other people have to adjust as well?

Reply:
So, I just wrote this change, but then both ('Far Shackles on Popoto' and 'Close Shackles on YOU') will pop up at the same time; do you think that's okay?

Reply:
Close Shackles on YOU (alert), Far Shackles on Popoto (info) [slight delay for tts?? or no tts??]

If you're fancy you could collect both and say "popoto far, poutini close" if it's not on you for either.

Reply:
This has been updated; should something fancy happy for the tts? Like, if you got either of the alerts then just no tts on the infoText?

Reply:
Yeah! I think if you got either, then no tts.  I think you could drop the delay if you did that.

Reply:
Delay is unfortunately necessary for it to work; the cast finishes before the buffs go out; otherwise we can queue off of one of the buffs going out with a 1s delay if that's more obvious

Reply:
Could do it without a delay and trigger on when you have collected both buffs?